apps/heureka/src/App.tsx

```diff
@@ -4,6 +4,7 @@
  */
 
 import React from "react"
+// @ts-expect-error TS(2792): Cannot find module './styles.scss?inline'. Did you... Remove this comment to see the full error message
```

Comment:
add ` "vite-plugin-svgr/client"` to the types in tsconfig.json and remove this line

Reply:
yeah we need to [reference](https://github.com/cloudoperators/juno/blob/326b01e63be27508844fa9bc187f474bead43206/apps/supernova/src/types/index.d.ts#L6) `vite/client` types like we did in `supernova`.